integration-cli/docker_utils.go

```diff
@@ -679,6 +679,22 @@ func dockerCmdInDirWithTimeout(timeout time.Duration, path string, args ...strin
 	return out, status, err
 }
 
+func findContainerExitCode(c *check.C, name string, vargs ...string) (string, int) {
+	args := append(vargs, "inspect", "--format='{{ .State.ExitCode }} {{ .State.Error }}'", name)
+	cmd := exec.Command(dockerBinary, args...)
+	out, _, err := runCommandWithOutput(cmd)
+	if err != nil {
+		c.Fatal(err, out)
+	}
+	// return 127 for Windows build testing
```

Comment:
*  From master, I also get a State.ExitCode of -1.  However, I get output of **1** from `echo $?` 
*  From my branch, I get a State.ExitCode of 127 (Windows and Linux CLI) but with Windows CLI the  exit is still 1, while with Linux the exit is 127.  
*  What I really don't understand (yet) is why  with Windows CLI  exit code and State.ExitCode both are 126 in the case of `docker run busybox /etc' when the error is "permission denied" from contained cmd.